static/js/message_list_view.js

```diff
@@ -201,9 +201,16 @@ MessageListView.prototype = {
     },
 
     _add_msg_edited_vars: function (message_container) {
+        // This adds variables to message_container object which calculate bools for
+        // checking position of "(EDITED)" label as well as the edited timestring
```

Comment:
Please move the definitions from the commit message into a comment that lives here.